api/src/main/java/org/apache/cloudstack/api/command/user/template/ListTemplatesCmd.java

```diff
@@ -99,7 +99,7 @@
     public EnumSet<ApiConstants.DomainDetails> getDetails() throws InvalidParameterValueException {
         EnumSet<ApiConstants.DomainDetails> dv;
         if (CollectionUtils.isEmpty(viewDetails)) {
-            dv = EnumSet.of(ApiConstants.DomainDetails.min);
+            dv = EnumSet.of(ApiConstants.DomainDetails.all);
```

Comment:
@nvazquez any particular reason, we are listing all details by default? We can set it min and call API with details=all in UI when needed like DeployVM or template view

Reply:
@nvazquez can we use `min` when we don't want details by default, otherwise return all details like old behaviour?